addons/nuxeo-datadog-reporter/src/main/resources/OSGI-INF/datadog-contrib.xml

```diff
@@ -8,6 +8,27 @@
       <host>${datadog.host:=}</host>
       <pollInterval>${datadog.pollinterval:=60}</pollInterval>
       <tags>${datadog.tags:=nuxeo}</tags>
+
+      <filter method="substring">
+        <includes>
+          <include>jvm.</include>
+          <include>nuxeo.</include>
+        </includes>
+        <excludes>
+          <exclude>jvm.useless</exclude>
```

Comment:
Is this something real?

Reply:
not sure but there is no more addons/nuxeo-datadog module